samza-test/src/test/scala/org/apache/samza/test/harness/AbstractKafkaServerTestHarness.scala

```diff
@@ -86,6 +86,7 @@ abstract class AbstractKafkaServerTestHarness extends AbstractZookeeperTestHarne
     // if the test case requires setting up a cluster ACL,
     // then it needs to be implemented.
     setClusterAcl.foreach(_.apply())
+    servers.foreach(_.startup())
```

Comment:
qq: is it because of some change in TestUtils.createServer()? I am curious why we don't explicitly call startup() before?

Reply:
Hmm... thanks for catching this. I probably missed the fact that `TestUtils.createServer(_)` in the  line above already starts the server. I have removed this line in the latest patch.